docs/extensions/fares-v2.md

```diff
@@ -0,0 +1,82 @@
+| Discussion         | [Cross Dataset Namespaces](https://docs.google.com/document/d/1AtaiD_RClXYGbo5oBJBlHSoFJea2FQ3nZeQlX1WKTdA/edit)                                | Ito World                        | None specific to proposal — impacts concepts outside of fares in GTFS | It’s impractical to merge all interrelated transit systems into 1 dataset. We need some way to represent relationships between different GTFS feeds.                                                              | Add namespaces.txt to define what external datasets are referred to within the feed.                                                                                         |
+| Discussion         | [GTFS-References](http://bit.ly/gtfs-references)                                                                                                | Elizabeth Sall, Cal-ITP          | fare_transfer_rules, transfers, stop_times, pathways, stops           | Same as the Cross Dataset Namespaces proposal. We need some way to represent relationships between different GTFS feeds.                                                                                          | Use references to identify when an entity (like stop, pathway, transfer, fare_transfer_rule) are shared across datasets.                                                     |
+| Discussion         | [fare_leg_join_rules.txt Proposal + Thoughts on “as_route”](https://bit.ly/gtfs-fares-v2-fare-leg-join-rules-proposal)                          | Brian Ferris, Google             | fare_leg_rules                                                        | The as_route field proposed in the full Fares V2 proposal does not sufficiently support complex multi-leg fare behaviors.                                                                                         | Add fare_leg_join_rules.txt as a more general mechanism for indicating when multiple consecutive legs should be considered as one for the purposes of fare calculation.      |
+| Discussion         | [Should fare transfers require that the same fare product is used on both legs?](https://docs.google.com/document/d/1zF5i03BFQi1kVBfN17aR7N1KRrpsHs3B9YjCzE8apJE/edit)                                                              | Nicholas Paun, Transit           | fare_leg_rules, fare_products                                         | It is currently ambiguous in the spec if the same fare_product must be used on both legs of a fare_transfer_rule in order for the transfer to be accepted.                                                        | Add 1 field to clearly define if the transfer requires the same fare_product or not.                                                                                         |
+
+## History
```

Comment:
To go along the lines of the [GTFS Revision History](https://gtfs.org/schedule/process/#revision-history), we could display these with the most recent on top.

Reply:
Right now they're grouped roughly by problem they're trying to solve. I can't fully order by chronology because some of the Last Updates timestamps aren't publicly viewable.